lib/oli_web/live/users/user_actions.ex

```diff
@@ -26,6 +27,42 @@ defmodule OliWeb.Users.Actions do
 
     ~H"""
     <div>
+      <div class="form-group">
+        <label for="reset_link_input">Generate Reset Password Link</label>
+        <div class="input-group" style="max-width: 100%;">
+          <input
+            readonly
+            type="text"
+            id="invite-link-1"
+            class="form-control"
+            placeholder="Section Invite Link"
```

Comment:
Change "Section Invite Link" for the aria-label to "Password Reset Link Text Input" and since this is a read only input, simply remove the `placeholder` attribute altogether.